test/test_sse_channel.py

```
@@ -2,7 +2,8 @@
 
 from unittest import IsolatedAsyncioTestCase
 from eric_sse.prefabs import SSEChannel
-from eric_sse.entities import MessageQueueListener, Message
+from eric_sse.entities import Message
+from eric_sse.listener import MessageQueueListener
```

Ruff (0.11.9)

`eric_sse.listener.MessageQueueListener` imported but unused

Remove unused import: `eric_sse.listener.MessageQueueListener`

(F401)